tests/unit/test_order_request.py

```diff
@@ -222,13 +222,21 @@ def test__tool():
         _ = order_request._tool('notsupported', 'jsonstring')
 
 
-def test_clip_tool(geom_geojson, point_geom_geojson):
+def test_clip_tool_polygon(geom_geojson):
     ct = order_request.clip_tool(geom_geojson)
     expected = {'clip': {'aoi': geom_geojson}}
     assert ct == expected
 
-    with pytest.raises(exceptions.GeoJSONError):
-        _ = order_request.clip_tool(point_geom_geojson)
+
+def test_clip_tool_multipolygon(multipolygon_geom_geojson):
+    ct = order_request.clip_tool(multipolygon_geom_geojson)
+    expected = {'clip': {'aoi': multipolygon_geom_geojson}}
+    assert ct == expected
+
+
+def test_clip_tool_invalid(point_geom_geojson):
+    with pytest.raises(exceptions.ClientError):
+        order_request.clip_tool(point_geom_geojson)
```

Comment:
Could you explain what this last test is doing?

Also, do we have a practice of added short comments/descriptions in the code for our tests? Most of these tests made sense to me, but it would be nice if we could provide brief descriptions to make them easier to contextualize when we revisit them.

Overall, it looks great and will be a really useful feature for users.

Reply:
yeah I can add that! I just pushed the change